ansible_base/lib/utils/models.py

```diff
@@ -7,7 +7,6 @@
 from django.conf import settings
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import AbstractUser
-from django.db import models
```

Comment:
Checks are failing because this is used in type hints.

And also, I don't believe this is a problem, here's how you test: just run `python` by itself (without DJANGO_SETTINGS_MODULE set) and in the interpreter try these imports. The import `from django.db import models` works, but the import `from django.contrib.auth.models import AbstractUser` throws an error.